llama-index-integrations/embeddings/llama-index-embeddings-yandexgpt/pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = true
+import_path = "llama_index.embeddings.yandexgpt"
+
+[tool.llamahub.class_authors]
+VoyageEmbedding = "llama-index"
```

Comment:
Should be `YandexGPTEmbedding` and feel free to replace `llama-index` with your github username.